go.mod

```diff
@@ -14,8 +15,8 @@ require (
 	github.com/mitchellh/hashstructure v1.0.0
 	github.com/openshift/api v3.9.1-0.20190924102528-32369d4db2ad+incompatible
```

Comment:
you might wan to fix this!

Reply:
@Bowenislandsong thanks for noticing it! I've changed it to the tag being used by `openshift/oc` currently.